deep-security-agent/tasks/activate-linux.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+# *********************************************************************
+# * Activate the Deep Security Agent (For Linux only)
+# *********************************************************************
+---
+# *********************************************************************
+# * Check if DSA installed properly
+# *********************************************************************
+- debug:
+    msg: "Check if DSA installed properly."
+
+- stat:
+    path: /opt/ds_agent/dsa_query
+  register: dsaQuery
+
+- fail:
+    msg: 
+        - "Could not find command /opt/ds_agent/dsa_query to query DSA certificate."
+        - "Please make sure Deep Security Agent has been installed"
+  when: dsaQuery.stat.size is not defined 
```

Comment:
Why not using .stat.exists?

Reply:
Changed to .stat.exists to make it more straight-forward.